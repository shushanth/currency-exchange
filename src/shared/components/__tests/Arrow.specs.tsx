import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import { CEArrow } from '../index';

describe('<CEArrow />: snapshots', () => {
  test('tree snapshots match', () => {
    const arrowTree = renderer.create(<CEArrow />).toJSON();
    expect(arrowTree).toMatchSnapshot();
  });
});

describe('Component: <CEArrow />', () => {
  let CEArrowComponent: any;
  beforeEach(() => {
    CEArrowComponent = mount(<CEArrow />);
  });

  test('should render properly with default props', () => {
    expect(CEArrowComponent.length).toBeDefined();
    expect(CEArrowComponent.length).toBe(1);
  });
  test('should render with two div elements with default proper', () => {
    expect(CEArrowComponent.find('div').length).toBeDefined();
    expect(CEArrowComponent.find('div').length).toBe(2);
  });
});
