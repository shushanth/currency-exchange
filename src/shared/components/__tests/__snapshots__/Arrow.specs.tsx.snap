// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CEArrow />: snapshots tree snapshots match 1`] = `
<div
  className="sc-AxiKw hQHhPG"
>
  <div
    className="sc-AxhCb fGhDwl"
  />
</div>
`;
